### DISCUSSION TITLE
绳网邮箱通知怎么关？

### DISCUSSION THREAD

**Discussion by YeMengXue** [share121/inter-knot#1006] [其他]
想知道用邮箱登录绳网后，回复提醒之类的通知怎么关？

**Comment by wwwwsn147** [comment]
把邮箱删了(雾

**Comment by nahida0-o** [comment]
你可以翻一下之前的帖子，有人发了教程，实测管用。

**Comment by Drs1145** [comment]
点开邮箱，有一个unsubscribe点一下就好了

**Comment by lllliufeng** [comment]
Notifications（盒子图标）→Manage notifacations→Notification settings→Participating, @mentions and custom的Notify me把Email勾掉

**Comment by Buer-Nahida** [comment]
见https://github.com/share121/inter-knot/discussions/512

**Comment by Gmadelyn7** [comment]
点击discussion页面右侧的unsubscribe
